src/main/java/com/thealgorithms/maths/Factorial.java

```diff
@@ -1,15 +1,6 @@
 package com.thealgorithms.maths;
 
 public class Factorial {
```

Comment:
I think it does not make sense to have any objects of the class `Factorial` - one can make a _proper_ utility  class of it. What do you think about the following change:
```suggestion
public final class Factorial {
    private Factorial() {
    }
```

Reply:
Thanks for the suggestion. But I think it is not necessary at all.

Reply:
@vil02  While the usage of final itself is good, mostly we don't use it at the moment. I think we can make a transition everywhere at once to keep code consistent

Reply:
@siriak The main part of this suggestion was to explicitly define the default constructor of `Factorial` as private. The `final` is just a nice small addition.

Reply:
I see now, I missed this at first. Anyway, feel free to raise a PR with the changes you propose, and we can continue discussion there

Reply:
cf. #4267.